distributed/client.py

```diff
@@ -3022,6 +3022,7 @@ def _get_computation_code(
             stacklevel = stacklevel if stacklevel > 0 else 1
 
         code: list[str] = []
+        is_ipython_idx = None
```

Comment:
nit: in my mind `is_foo` indicates a bool

Reply:
https://github.com/dask/distributed/pull/7788/commits/4bfbbff7d0ad9c0eeee7007249f3a07481e1c662